packages/xrpl/test/models/payment.test.ts

```diff
@@ -258,4 +375,21 @@ describe('Payment', function () {
       'PaymentTransaction: tfPartialPayment flag required with DeliverMin',
     )
   })
+
+  it(`verifies deliver_max alias in PaymentTransactions`, function () {
+    assert.doesNotThrow(() => validatePayment(paytxn1))
+    assert.doesNotThrow(() => validate(paytxn1))
+
+    assert.doesNotThrow(() => validatePayment(paytxn2))
+    assert.doesNotThrow(() => validate(paytxn2))
+
+    assert.doesNotThrow(() => validatePayment(paytxn3))
+    assert.doesNotThrow(() => validate(paytxn3))
+
+    assert.throws(() => validatePayment(paytxn4))
+    assert.throws(() => validate(paytxn4))
+
+    assert.throws(() => validatePayment(paytxn5))
+    assert.throws(() => validate(paytxn5))
+  })
```

Comment:
Each different test should be separated into its own `it()` function.

Reply:
ok, I've addressed this comment in a8036ae7e54219003e03ed41ffb5bb347335eb30